pkg/printers/internalversion/printers.go

```diff
@@ -41,7 +41,6 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
```

Comment:
Also, that's where - for better or worse - we decided to store the node role labels.  There was some corruption during the move, but we should fix it up from this point i.e. add the labels that were removed, put it into a `pkg` directory if we don't want to reference a `cmd` directory